lexoid/core/parse_type/static_parser.py

```diff
@@ -80,11 +86,13 @@ def parse_static_doc(path: str, **kwargs) -> Dict:
     elif "wordprocessing" in file_type:
         return parse_with_docx(path, **kwargs)
     elif file_type == "text/html":
-        with open(path, "r") as f:
+        logger.debug(f"Parsing HTML file: {path}")
+        with open(path, "r", errors="ignore") as f:
             html_content = f.read()
             return html_to_markdown(html_content, kwargs["title"])
     elif file_type == "text/plain":
-        with open(path, "r") as f:
+        logger.debug(f"Parsing plain text file: {path}")
+        with open(path, "r", errors="ignore") as f:
```

Comment:
One thing we might wanna keep in mind that `ignore` might lead to info loss.
Maybe for now `ignore` is fine. If we start running into use cases where info loss becomes critical, we can use `replace` or `backslashreplace`?